circle.yml

```diff
@@ -151,6 +151,49 @@ commands:
           name: Restore all node_modules to proper workspace folders
           command: node scripts/circle-cache.js --action unpack
 
+  restore_cached_system_tests_deps:
+    steps:
+      - run:
+          name: Generate Circle Cache key for system tests
+          command: yarn workspace @tooling/system-tests cache:key > system_tests_cache_key
```

Comment:
Discussed this with @tgriesser originally, but decided to split this out because the `system-tests/projects` cache key is way simpler to generate than the `circle-cache.js` key. It's a one-liner, versus the somewhat complicated logic in `circle-cache` to generate a key. This way we can keep `circle-cache.js`'s specific purpose as clear as possible.